src/api/index.js

```diff
@@ -127,6 +127,46 @@ export function createPipelineRun({ namespace, payload }) {
   return post(uri, payload);
 }
 
+export function createPipelineRunAtProxy({
+  namespace,
+  pipelineRef,
```

Comment:
can we call this `pipelineName` as 'ref' is specific to the API / CRD structure and not something a consumer should really need to be aware of

Reply:
Yep, sounds good.